version/version.go

```diff
@@ -1,5 +1,30 @@
 package version
 
+import (
+	"fmt"
+	"runtime"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+
+	sdkVersion "github.com/operator-framework/operator-sdk/version"
+)
+
+var log = logf.Log.WithName("version")
+
+//var needs to be used instead of const for ldflags
 var (
-	Version = "0.0.1"
+	Version            = "0.0.1"
+	GoVersion          = fmt.Sprintf("%s %s/%s", runtime.Version(), runtime.GOOS, runtime.GOARCH)
+	OperatorSDKVersion = sdkVersion.Version
```

Comment:
Do we really need these global variables? I prefer just having them inline in the `Print()` function like it was originally.